osf/management/commands/manage_switch_flags.py

```diff
@@ -1,38 +1,34 @@
 # -*- coding: utf-8 -*-
 import logging
+import yaml
 
 from django.core.management.base import BaseCommand
 
-from osf.features import switches, flags
-from waffle.models import Flag, Switch
-
 logger = logging.getLogger(__name__)
 
-def manage_waffle(delete_waffle=False):
-    file_switches = list(switches.values())
-    current_switches = Switch.objects.values_list('name', flat=True)
 
-    add_switches = set(file_switches) - set(current_switches)
-    for switch in add_switches:
-        Switch.objects.get_or_create(name=switch, defaults={'active': False})
-        logger.info('Adding switch: {}'.format(switch))
+def manage_waffle(delete_waffle=False):
+    from django.apps import apps
 
-    file_flags = list(flags.values())
-    current_flags = Flag.objects.values_list('name', flat=True)
+    Flag = apps.get_model('waffle.Flag')
```

Comment:
nit: Add note that import is occurring inline to enable use as post_migrate_signal